packages/components/src/FormField/components/ClearAction.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./ClearAction.css";
+import { Icon } from "../../Icon";
+
+interface ClearActionProps {
+  /**
+   * Click handler
+   */
+  readonly onClick: () => void;
+}
+
+export function ClearAction({ onClick }: ClearActionProps): JSX.Element {
```

Comment:
As a followup to this PR it would be nice to swap this component into [ComboboxContentSearch](https://github.com/GetJobber/atlantis/blob/da2879c62e6740a1d37263b1ac37444f903e8a9e/packages/components/src/Combobox/components/ComboboxContent/ComboboxContentSearch/ComboboxContentSearch.tsx)